components/log-viewer-webui/common/index.ts

```
@@ -0,0 +1,75 @@
+interface ServerToClientEvents {
+    "collection::find::update": (respArgs: {
+        queryId: number;
+        data: object[];
+    }) => void;
+}
+
+/**
+ * Empty but required by Socket IO.
+ */
+interface InterServerEvents {
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)